mcsema/BC/Instruction.cpp

```diff
@@ -245,6 +245,10 @@ llvm::Value *InstructionLifter::GetAddress(const NativeXref *cfg_xref) {
     // `__gmon_start__`, which is likely to be an ELF thunk. So we throw in
     // an extra layer of indirection here.
     //
+#if 0
```

Comment:
Do we have a better solution? 

Since this is in `#if 0`, can the code just be deleted?

Reply:
I need to discuss it with @pgoodman. The extra check was causing garbage value for the destructor function `_ZNSt12out_of_rangeD1Ev`.

Reply:
Hrm lets move discussion of this to Slack. I don't recall enough of the context right now.